app/views/search/_granule_list.html.erb

```diff
@@ -100,19 +100,21 @@ Delimiters: Separate multiple granule IDs by space, comma, or new line."/>
                 <a href="#" class="button icon-button master-overlay-forward" data-master-overlay-forward="Granules" data-bind="click: $parent.showGranuleDetails" title="View granule details">
                   <i class="fa fa-info-circle"></i>
                 </a>
+
                 <!--ko if: dataLinks().length == 1-->
                   <a class="button icon-button direct-download" title="Download single granule data" target="_blank" data-bind="visible: !$parent.collection.cwic() && online_access_flag, attr: {href: dataLinks()[0].href}">
                     <i class="fa fa-download"></i>
                   </a>
                 <!--/ko-->
                 <!--ko if: dataLinks().length > 1-->
-                  <a href="#" data-toggle="dropdown" class="button icon-button" aria-expanded="false"><i class="fa fa-download"></i></a>
+                  <a href="#" data-toggle="dropdown" class="button icon-button" aria-expanded="false"><i class="fa fa-download"></i></a>    
```

Comment:
There's been extra space added at the end of this line - is there a purpose for it? Is this too nitpicky to comment?